carapp/security.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+import jwt
+from werkzeug.security import check_password_hash
+from carapp import app, db_models
+from flask import make_response, request
+from functools import wraps
+
+# app.config["SECRET_KEY"] = "thisissecret"
+
+
+def token_required(any_function):
```

Comment:
This one is cool. And I believe was cognitive :) Let's have a separate ticket to use a third-party library for the implementation of the Bearer auth.

Reply:
Yes sure, cool!